R/mod_fft.R

```diff
@@ -43,7 +43,13 @@ mod_fft_server <- function(id, filter_data){
       
     graph_data <- reactive({
       
-      split_data_spc(filter_data()$unique_data, variable = "fft", chunks = 'monthly')
+      tryCatch({
```

Comment:
Why is this in a tryCatch block? What error does it throw and why?

Reply:
I guess I was meant to remove this. I added it when trying some things

Reply:
Sorry, just figured this out. the `trycatch()` was in the wrong part. the error arises when the selected date range is not enough to do `date = as.Date(cut(date, "month"))` in `split_data_spc()`